tests/unit/useWeekTemplates.test.ts

```
@@ -0,0 +1,417 @@
+    eq: vi.fn().mockReturnThis(),
+    order: vi.fn().mockReturnThis(),
+    single: vi.fn().mockResolvedValue({ data: mockTemplate, error: null }),
+  };
+
+  // By default, order() resolves as the terminal call for queries
+  // For chained .order().order(), the second order is the terminal call
+  mockFromChain.order.mockImplementation(() => {
+    const chainable = {
+      ...mockFromChain,
+      // When order is the last in chain (for queries), resolve
+      then: (resolve: (value: { data: unknown[]; error: null }) => void) => {
```

Biome (2.4.4)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)